templates/extra/resources.html

```diff
@@ -0,0 +1,293 @@
+                                    <li><p class="content-sub"><a href={% url 'resources' %}#launch-4>Un-Successful</a></p></li>
+                                </ul>
+                            </li>
+                        </ul>
+                        <!-- Success -->
+                        <ul class="vertical menu content-box" data-accordion-menu>
+                            <li>
+                                <a class="content-header" href="overview-nested">Success</a>
+                                <ul class="menu vertical nested">
+                                    <li><p class="content-sub"><a href={% url 'resources' %}#success-1>Once Funded</a></p></li>
+                                    <li><p class="content-sub"><a href={% url 'resources' %}#success-2>Recording music</a></p></li>
+                                    <li><p class="content-sub"><a href={% url 'resources' %}#success-3>Copyrights</a></p></li>
+                                    <li><p class="content-sub"><a href={% url 'resources' %}#success-4>Distribution</a></p></li>
+                                    <li><p class="content-sub"><a href={% url 'resources' %}#success-5>Project Information</a></p></li>
+                                    <li><p class="content-sub"><a href={% url 'resources' %}#success-6>Once project is live</a></p></li>
+                                </ul>
+                            </li>
+                        </ul>
+                        <!-- Next Steps -->
+                        <ul class="vertical menu content-box" data-accordion-menu>
+                            <li>
+                                <a class="content-header" href="overview-nested">Next Steps</a>
+                                <ul class="menu vertical nested">
+                                    <li><p class="content-sub"><a href={% url 'resources' %}#next-1>Engaging Investors</a></p></li>
+                                    <li><p class="content-sub"><a href={% url 'resources' %}#next-2>Raising More</a></p></li>
+                                    <li><p class="content-sub"><a href={% url 'resources' %}#next-3>Another Campaign</a></p></li>
+                                    <li><p class="content-sub"><a href={% url 'resources' %}#next-4>Helping others</a></p></li>
+                                </ul>
+                            </li>
+                        </ul>
+                    </ul>
+                </div>
+            </div>
+            <div class="columns small-9 float-right" id="example1" data-something>
+                <div id="overview">
+                    <h3>overview</h3>
+                    <div class="break">
+                        <hr>
+                    </div>
+                    <div id="overview-1">
+                        <h4>What is PerDiem?</h4>
+                        <p>
+                        A platform where you can raise money for your music projects by having fans and people invest in your music. 
```

Comment:
nit: Indent all of these paragraphs in, or keep them on the same line. ie.
```html
<p>
    foo
</p>
```
or
```
<p>foo</p>
```
but not
```
<p>
foo
</p>
```